main.go

```diff
@@ -51,39 +54,64 @@ func main() {
 		return
 	}
 
+	c, err := parseConfig()
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	b, err := NewBuilder(c)
+	if err != nil {
+		log.Fatal(err)
+	}
+	defer b.Close()
+
+	wd, err := os.Getwd()
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	b.Watch(wd)
+}
+
+func parseConfig() (config, error) {
 	var c config
-	if len(cliCmds) > 0 {
-		c.Script = cliCmds
-	} else {
+
+	// if we have any cliCmds, set them to our build phase
+	c.Build = cliCmds
+
+	// if build phase is still empty try and find the snag.yml file
+	if len(c.Build) == 0 {
 		in, err := ioutil.ReadFile(".snag.yml")
 		if err != nil {
-			log.Fatal("Could not find '.snag.yml' in your current directory")
+			return c, errors.New("Could not find '.snag.yml' in your current directory")
 		}
 
 		if err := yaml.Unmarshal(in, &c); err != nil {
-			log.Fatalf("Could not parse yml file. %s\n", err)
+			return c, fmt.Errorf("Could not parse yml file. %s\n", err)
 		}
-
 	}
 
-	if len(c.Script) == 0 {
-		log.Fatal("You must specify at least 1 command.")
+	// if both script and build are specified
+	// blow up and tell the user to use build
+	if len(c.Script) != 0 && len(c.Build) != 0 {
+		return c, errors.New("Cannot use 'script' and 'build' together. The 'script' tag is deprecated, please use 'build' instead.")
 	}
 
-	if verbose {
-		c.Verbose = verbose
+	// if script has something, tell the user it's deprecated
+	// and set whatever its contents are to build
+	if len(c.Script) != 0 {
+		fmt.Println("The use of 'script' in the yaml file has been deprecated and will be removed in the future.")
```

Comment:
293b852 - Ideally it'd be nice to have it in color but the coloring stuff is inside the vow package. I'd rather wait until we address #21 before we start exporting stuff inside the vow package

<img width="717" alt="screen shot 2015-12-23 at 12 35 41 pm" src="https://cloud.githubusercontent.com/assets/2306786/11981566/dfb283ec-a971-11e5-83e3-6eab9664924e.png">